src/components/shared/Links/links.tsx

```diff
@@ -0,0 +1,19 @@
+interface ButtonLargeProps {
+  text: string;
+  url: string;
+}
+
+const Link = ({ text, url }: ButtonLargeProps) => {
+  return (
+    <a
+      href={url}
+      className={
+        'font-semibold text-links disabled:text-dark-grey hover:underline focus:outline-focus focus:outline-dashed active:text-active active:outline-none'
```

Comment:
Is it necessary to add those pseudo-classes? For example, `hover: underline`, I think that's the default behaviuor